### ISSUE TITLE
Current dev-master won't save new models

### ISSUE THREAD

**Issue by mneil** [FriendsOfCake/crud-view#80]
I just updated crud-view hoping to fix some other issues and found that I can no longer save new models. I have a test application setup just pulling the latest version of cake3 and crud-view. When trying to add a new entry to my "blog_entries" table it never saves. Instead it just redirects me back to the add page with the query string filled with the data I had in it.

To replicate update crud-view. Try to add a new model.

**Comment by mneil** [comment]
Rescinding this issue for now. Looks like it's something related to auth component. I noticed a 302 happening once I started logging the post requests and traced it back to auth.

**Comment by mneil** [comment]
In case anyone else runs into this issue it had to do with the [Search component](https://github.com/FriendsOfCake/search). I disabled it and all of a sudden my saves worked. I thought it was authentication because I disabled all components except those required by crud-view and it worked. But afterwards I added back the auth component and it still worked. It wasn't until I enabled the search component that saving stopped working.
